app/tests/integration/conftest.py

```diff
@@ -49,30 +49,46 @@ def _verify_url(request, base_url):
         session.get(base_url, verify=False)
 
 
+@pytest.fixture(name="experiment_type")
+def fixture_experiment_type():
```

Comment:
Hi @jrbenny35! 👋 

If `experiment_type` is defined in modules containing tests that depend on this to be set, you should be able to delete this here. Otherwise you might be seeing `KeyError` exceptions in `experiment_overview_info` instead of a pytest error.

Reply:
So I left it here because the HTML validation tests don't need an experiment type. I guess I could just split the ```fixture_experiment_info``` into a default and a ```variables``` selected one